pkg/get/get_test.go

```diff
@@ -5027,6 +5027,53 @@ func Test_DownloadKubeconform(t *testing.T) {
 	}
 }
 
+func Test_DownloadKubeLinter(t *testing.T) {
+	tools := MakeTools()
+	name := "kube-linter"
+	version := "v0.6.4"
+
+	tool := getTool(name, tools)
+
+	tests := []test{
+		{
+			os:      "darwin",
+			arch:    arch64bit,
+			version: version,
+			url:     "https://github.com/stackrox/kube-linter/releases/download/v0.6.4/kube-linter-darwin",
+		},
+		{
+			os:      "linux",
+			arch:    arch64bit,
+			version: version,
+			url:     "https://github.com/stackrox/kube-linter/releases/download/v0.6.4/kube-linter-linux",
+		},
+		{
+			os:      "linux",
+			arch:    archARM64,
```

Comment:
looks like `kube-linter` doesn't support ARM at all, therefore this test is redundant. 

I think that we need to hardcode the arch64 as the only supported architecture in the template and remove the redundant test. @alexellis what are your thoughts here?

Reply:
You are right I will remove it ;)